include/libpmemfile-posix.h

```diff
@@ -311,6 +311,13 @@ int pmemfile_ftruncate(PMEMfilepool *, PMEMfile *file, off_t length);
 int pmemfile_fallocate(PMEMfilepool *, PMEMfile *file, int mode,
 		off_t offset, off_t length);
 
+static inline int
```

Comment:
Please move the definition to C file. If we'll need to change something in the implementation (like add or remove some checks) it will not be possible with static inline impl.